class.jetpack-connection-banner.php

```diff
@@ -301,6 +307,66 @@ class="dops-button is-primary jp-banner__alt-connect-button">
 		<?php
 	}
 
+	/**
+	 * Renders the license-away version of the connection banner.
+	 *
+	 * @since 9.1.0
+	 */
+	public function render_license_aware_banner() {
+		?>
+		<div id="message" class="updated jp-wpcom-connect__container">
+			<div class="jp-wpcom-connect__inner-container">
+				<div class="jp-wpcom-connect__content-container">
+					<!-- slide 1: intro -->
+					<div class="jp-wpcom-connect__slide jp-wpcom-connect__slide-one jp__slide-is-active">
+
+						<div class="jp-wpcom-connect__content-icon jp-connect-illo">
+							<?php echo ( new Logo() )->render(); ?>
+							<img
+								src="<?php echo esc_url( plugins_url( 'images/jetpack-powering-up.svg', JETPACK__PLUGIN_FILE ) ); ?>"
+								class="jp-wpcom-connect__hide-phone-and-smaller"
+								alt="
+								<?php
+								esc_attr_e(
+									'Jetpack premium services offer even more powerful performance, security, and revenue tools to help you keep your site safe, fast, and help generate income.',
+									'jetpack'
+								);
+								?>
+								"
+								height="auto"
+								width="225"
+								/>
+						</div>
+
+						<div class="jp-wpcom-connect__slide-text">
+							<h2 class="jp-wpcom-connect__quest"><?php esc_html_e( 'Your Jetpack purchase needs completion! Please set up the plugin for your subscription.', 'jetpack' ); ?></h2>
+
+							<p>
+								<?php
+								esc_html_e(
+									'Jetpack offers security, performance, and marketing tools made for WordPress sites by the WordPress experts. Set up Jetpack to enable new features for this site; don\'t let your subscription go to waste!',
+									'jetpack'
+								);
+								?>
+							</p>
+
+							<div class="jp-banner__button-container">
+								<span class="jp-banner__tos-blurb"><?php jetpack_render_tos_blurb(); ?></span>
+								<a
+									href="<?php echo esc_url( $this->build_connect_url_for_slide( '72' ) ); ?>"
```

Comment:
Should this be `90` instead of `72`, since this is a new banner for this release?

```suggestion
									href="<?php echo esc_url( $this->build_connect_url_for_slide( '90' ) ); ?>"
```